### PR TITLE
Remove row from Pro vs OSS table

### PR DESCRIPTION
**Pull request by sinat101**
This is not actually accurate. Both OSS and Pro support GitLab and SARIF (GitHub) output.

# Thanks for improving Semgrep Docs 😀

### Please ensure

- [x] A subject matter expert (SME) reviews the content
- [ ] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team
- [x] Redirects are added if the PR changes page URLs
- [x] If you have changed any header tag links (doc/#this-kind-of-anchor), update all instances of that link

### CODE REVIEW

**Comment by s-santillan** [comment]
Just checking - can Semgrep OSS (i.e. through `semgrep scan --sarif`) send the scan results to GHAS through an artifact? @sinat101

**Comment by sinat101** [comment]
> Just checking - can Semgrep OSS (i.e. through `semgrep scan --sarif`) send the scan results to GHAS through an artifact? @sinat101

@s-santillan can you clarify what you mean by "through an artifact"? both OSS and Pro just generate a .sarif file that the user then has to somehow upload to GHAS

**Comment by s-santillan** [comment]
@sinat101 Yeah, an **artifact** is a file produced by the job. From the sound of it, yes both OSS and proprietary support it. https://github.com/actions/upload-artifact . Let me check the doc if we want to keep the row but :heavy_check_mark: both OSS and proprietary. Thank you.

**Comment by sinat101** [comment]
@s-santillan ahh right, that option works as well!

File: docs/semgrep-pro-vs-oss.md [hunk 1]
```diff
@@ -73,7 +73,6 @@ The following table displays various notification channels and reporting feature
 | --------------------------------------------------------------------------------------------------------------- | ----------- | ----------------- |
 | [Centralized management of scan results (triage, remediation, fine-tuning noisy rules)](/semgrep-code/policies) | --          | ✔️                |
```
Comment (s-santillan) [inline docs/semgrep-pro-vs-oss.md:74]: | Send scan results to GitLab SAST and GitHub Advanced Security | ✔️ | ✔️ |

File: docs/semgrep-pro-vs-oss.md [hunk 2]
```diff
@@ -73,7 +73,6 @@ The following table displays various notification channels and reporting feature
 | --------------------------------------------------------------------------------------------------------------- | ----------- | ----------------- |
```
Comment (s-santillan) [inline docs/semgrep-pro-vs-oss.md:73]:
```suggestion
| --------------------------------------------------------------------------------------------------------------- | ----------- | ----------------- |
| Send scan results to GitLab SAST and GitHub Advanced Security | ✔️ | ✔️ |
```

Comment (s-santillan) [reply docs/semgrep-pro-vs-oss.md:73]: Suggest adding it back but with a checkmark on both? @sinat101

**Review by s-santillan — APPROVED** [review]

**Pull request merged**